pkg/kopia/repository/client.go

```diff
@@ -64,6 +64,7 @@ func ConnectToAPIServer(
 		CachingOptions: content.CachingOptions{
 			CacheDirectory:              kopia.DefaultClientCacheDirectory,
 			ContentCacheSizeLimitBytes:  int64(contentCacheMB << 20),
+			ContentCacheSizeBytes:       int64(contentCacheMB << 20),
```

Comment:
```suggestion
			ContentCacheSizeBytes:       int64(contentCacheMB << 20),
			MetadataCacheSizeBytes:      int64(metadataCacheMB << 20),
```
Let's set both for now. No harm